test/validation/api/classification/odp_classification_test_pmr.c

```diff
@@ -1580,9 +1579,6 @@ static void classification_test_pmr_term_sctp(bool is_dport)
 	CU_ASSERT(retqueue == queue);
 	recvpool = odp_packet_pool(pkt);
 	CU_ASSERT(recvpool == pool);
-	sctp = (odph_sctphdr_t *)odp_packet_l4_ptr(pkt, NULL);
-	CU_ASSERT(sctp->tag == 0x11223344);
-	odp_packet_free(pkt);
```

Comment:
I think you are referring to this line. You are right, odp_packet_free() is needed. I somehow screwed this up when splitting this to a separate commit. Thanks.

Note that my suggestion was to consider this commit as a review comment to your PR so that you could add these changes and then I could just delete this commit from this PR.